tensorflow/c/experimental/filesystem/filesystem_interface.h

```diff
@@ -84,22 +84,28 @@ typedef struct TF_TransactionToken {
 } TF_TransactionToken;
 
 typedef struct TF_Filesystem_Option_Value {
-  int type_tag;
-  int num_values;
```

Comment:
We should still keep this to allow multiple values for the same key (or a vector of values)

Reply:
@mihaimaruseac Updated.